src/sonic-host-service/sonic-hostservice.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=SONiC Host Service
+
+[Service]
+Type=dbus
+BusName=org.SONiC.HostService
+
+ExecStart=/usr/bin/python3 -u /usr/lib/sonic_host_service/sonic_host_server.py
+
+Restart=on-failure
+RestartSec=3
```

Comment:
Can we increase  this to 10sec?
If in  case of any persistent failure, say incorrect .py file in host modules, every 3 secs can be a heavy load on control plane, as  it would be continuous restart.

Reply:
Should be OK.

Reply:
Does it mean, we are switching to 10secs?

Reply:
Yes, I will update this in my next commit.